test/loading-bar-interceptor.coffee

```diff
@@ -165,6 +165,47 @@ describe 'loadingBarInterceptor Service', ->
     expect(cfpLoadingBar.status()).toBe 1
     $timeout.flush()
 
+  it 'should not increment if the response is cached with different params', inject (cfpLoadingBar) ->
+    return unless angular.version.major >= 1 && angular.version.minor >= 4
+
+    $httpBackend.expectGET(endpoint).respond response
+    $http.get(endpoint, cache: true).then (data) ->
+      result = data
+
+    expect(cfpLoadingBar.status()).toBe 0
+    $timeout.flush()
+    $timeout.flush()
```

Comment:
Good question.
I must admit that I stupidly copied the structure from existing tests.